doc/getting_started/installation_linux.rst

```diff
@@ -65,65 +70,77 @@ Install Requirements and Dependencies
 
-The Clear Linux focus is on *native* performance and security and not
-cross-compilation. For that reason it uniquely exports by default to the
-:ref:`environment <env_vars>` of all users a list of compiler and linker
-flags. Zephyr's CMake build system will either warn or fail because of
-these. To clear the C/C++ flags among these and fix the Zephyr build, run
-the following command as root then log out and back in:
+      The Clear Linux focus is on *native* performance and security and not
+      cross-compilation. For that reason it uniquely exports by default to the
+      :ref:`environment <env_vars>` of all users a list of compiler and linker
+      flags. Zephyr's CMake build system will either warn or fail because of
+      these. To clear the C/C++ flags among these and fix the Zephyr build, run
+      the following command as root then log out and back in:
 
-.. code-block:: console
+      .. code-block:: console
 
-   # echo 'unset CFLAGS CXXFLAGS' >> /etc/profile.d/unset_cflags.sh
+         echo 'unset CFLAGS CXXFLAGS' >> /etc/profile.d/unset_cflags.sh
 
-Note this command unsets the C/C++ flags for *all users on the
-system*. Each Linux distribution has a unique, relatively complex and
-potentially evolving sequence of bash initialization files sourcing each
-other and Clear Linux is no exception. If you need a more flexible
-solution, start by looking at the logic in
-``/usr/share/defaults/etc/profile``.
+      Note this command unsets the C/C++ flags for *all users on the
+      system*. Each Linux distribution has a unique, relatively complex and
+      potentially evolving sequence of bash initialization files sourcing each
+      other and Clear Linux is no exception. If you need a more flexible
+      solution, start by looking at the logic in
+      ``/usr/share/defaults/etc/profile``.
 
-Arch Linux:
+   .. group-tab:: Arch Linux
 
-.. code-block:: console
+      .. code-block:: console
 
-   sudo pacman -S git cmake ninja gperf ccache dfu-util dtc wget \
-       python-pip python-setuptools python-wheel tk xz file make
+         sudo pacman -S git cmake ninja gperf ccache dfu-util dtc wget \
+             python-pip python-setuptools python-wheel tk xz file make
 
-**CMake version 3.13.1 or higher is required**. Check what version you have by
+CMake
+=====
+
+CMake version 3.13.1 or higher is required. Check what version you have by
 using ``cmake --version``. If you have an older version, there are several ways
 of obtaining a more recent one:
 
-* Use ``pip3`` (see :ref:`python-pip` for more details):
+* Download and install a packaged cmake from the CMake project site:
 
   .. code-block:: console
 
-     pip3 install --user cmake
+     cd ~
+     wget https://github.com/Kitware/CMake/releases/download/v3.15.3/cmake-3.15.3-Linux-x86_64.sh
+     chmod +x cmake-3.15.3-Linux-x86_64.sh
+     sudo ./cmake-3.15.3-Linux-x86_64.sh --skip-license --prefix=/usr/local
```

Comment:
I think some comments both here and in the GSG saying "go update the other file if you change this" would ease maintainability.